app/scripts/components/sandbox/request/index.js

```diff
@@ -0,0 +1,140 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { useQueryClient } from '@tanstack/react-query';
+import { datasets } from 'delta/thematics';
+import { FormCheckable } from '@devseed-ui/form';
+import { Button } from '@devseed-ui/button';
+import { Toolbar } from '@devseed-ui/toolbar';
+
+import Constrainer from '$styles/constrainer';
+import { PageMainContent } from '$styles/page';
+import { requestStacDatasetsTimeseries } from './timeseries-data';
+
+const Wrapper = styled.div`
+  position: relative;
+  grid-column: 1 / -1;
+`;
+
+export default function SandboxRequest() {
+  const [selectedLayers, setSelectedLayers] = useState([]);
+  const [requestStatus, setRequestStatus] = useState([]);
+
+  /* eslint-disable-next-line */
+  console.log("Requets status", requestStatus);
+
+  const queryClient = useQueryClient();
+
+  const getData = (selectedLayers_) => {
+    const layers = Object.values(datasets)
+      .flatMap((d) => d.data.layers)
+      .filter((d) => selectedLayers_.includes(d.id));
+
+    setRequestStatus([]);
+    queryClient.cancelQueries(['analysis']);
+    const requester = requestStacDatasetsTimeseries({
+      layers,
+      queryClient
+    });
+
+    requester.on('data', (data, index) => {
+      setRequestStatus((dataStore) =>
+        Object.assign([], dataStore, {
+          [index]: data
+        })
+      );
+    });
+  };
```

Comment:
In the analysis page this will be achieved with a useEffect instead of a callback from a button click. This is just for demo purposes.